src/cpp/session/include/session/prefs/UserState.hpp

```diff
@@ -16,7 +16,8 @@
 #ifndef SESSION_USER_STATE_HPP
 #define SESSION_USER_STATE_HPP
 
-#define kUserStateFile "rstudio-state.json"
+#define kUserStateFileDesktop "rstudio-desktop.json"
```

Comment:
Does this change require a migration from existing users of v1.3 that already have rstudio-state.json ?

Reply:
Unfortunately it does. I've done that as a follow up change here: 10a684669d8816c9d3c30dce7b39715f395a2c26

Reply:
Okay great!